test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -960,6 +988,7 @@ void UcxConnection::established(ucs_status_t status)
     if (!_use_am && (status == UCS_OK)) {
         assert(_remote_conn_id != 0);
         UCX_CONN_LOG << "Remote id is " << _remote_conn_id;
+        print_addresses();
```

Comment:
can we print the addresses in same line with remote id, to reduce number of log messages?

Reply:
@yosefe We can, but not simple. We want to print address when ucp_ep_query status is UCS_OK, but print failure when status is not UCS_OK. using UCX_CONN_LOG to print log in single line can't support condition check. Or we need add more code to deal with it.
Here we only add one line of address log when connection estabilished, so maybe not so important to reduce it?

Reply:
@yosefe I fixed it.